docs/book/themes/themes.rst

```diff
@@ -0,0 +1,105 @@
+.. index::
+   single: Themes
+
+Themes
+======
+
+Theming is a method of customizing how your channels look like in Sylius. Each channel can have a different tqheme.
```

Comment:
Well described @gabiudrescu, I agree.

@TheMadeleine it's important to put this information in the docs, there are already questions about that.